include/crpropa/magneticField/turbulentField/TurbulentField.h

```diff
@@ -0,0 +1,119 @@
+#ifndef CRPROPA_TURBULENTFIELD_H
+#define CRPROPA_TURBULENTFIELD_H
+
+#include "crpropa/magneticField/MagneticField.h"
+#include <cmath>
+
+namespace crpropa {
+/**
+ * \addtogroup MagneticFields
+ * @{
+ */
+
+/**
+ @class TurbulenceSpectrum
+ @brief Defines the energy spectrum of turbulence
+ */
+class TurbulenceSpectrum : public Referenced {
+  private:
+	const double Brms; /**< Brms value of the turbulent field (normalization) */
+	const double sIndex; /**< Spectral index for the inertial range, for example
+	                  s=5/3 for Kolmogorov spectrum */
+	const double qIndex; /**< Spectral index for the injection range, for
+	                  example q=4 for 3D homogeneous turbulence */
+	const double lBendover;  /**< the bend-over scale */
+	const double lMin, lMax; /**< Min and Max scale of turbulence */
+
+  protected:
+	/**
+	Normalization for the above defined energy spectrum
+	*/
+	double spectrumNormalization() const {
+		return std::tgamma((sIndex + qIndex) / 2.0) /
+		       (2.0 * std::tgamma((sIndex - 1) / 2.0) *
+		        std::tgamma((qIndex + 1) / 2.0));
+	}
+
+  public:
+	/**
+	   @param Brms         root mean square field strength for generated field
+	 @param lMin	 Minimum physical scale of the turbulence
+	 @param lMax	 Maximum physical scale of the turbulence
+	   @param lBendover	   the bend-over scale
+	 @param sindex	 Spectral index of the energy spectrum in the inertial
+	 range
+	 @param qindex	 Spectral index of the energy spectrum in the energy
+	 range
+	                       Usually, you'd want to use s=5./3.
+	                       and q=4 here, which will be comparable
```

Comment:
Do we need this comment as the values are set as the default anyway?

Reply:
Right. Removed.